mule-migration-tool-library/src/main/java/com/mulesoft/tools/migration/library/tools/mel/nocompatibility/InboundPropertiesNoCompatibilityResolver.java

```diff
@@ -55,4 +55,9 @@ protected PropertyTranslator getTranslator() {
     return translator;
   }
 
+  @Override
+  protected String fallbackTranslation(String propertyToTranslate) throws Exception {
+    return translator.getAllTranslationsForAllSourceTypes().get(propertyToTranslate);
```

Comment:
wont this potentially return a translation for a sourcetype that is not in the app?

Reply:
Only in the case of a collision, that shouldn't happen much as there are only a few cases. We can handle collisions as a separate task, as doing it right may not be that simple.
If it's not a collision then it means there's a bug in the migrated app.